integrations/operator/controllers/resources/role_controller_test.go

```diff
@@ -94,13 +94,6 @@ allow:
 			expectedSpec: &types.RoleSpecV6{
 				Allow: types.RoleConditions{
 					Logins: []string{"ubuntu", "root"},
-					KubernetesResources: []types.KubernetesResource{
-						{
-							Kind:      types.KindKubePod,
-							Namespace: types.Wildcard,
-							Name:      types.Wildcard,
-						},
-					},
```

Comment:
Okay, I finally get it. Now we won't add the wildcard if the labels are unset. This removes noise but is not a breaking change as access was not possible because of the missing labels.